Assets/MixedRealityToolkit/Inspectors/Profiles/BaseMixedRealityToolkitConfigurationProfileInspector.cs

```diff
@@ -244,5 +245,50 @@ protected void CheckMixedRealityInputActions()
                 }
             }
         }
+
+        /// <summary>
+        /// Applies the most relevant settings of DataProviderConfigurations
+        /// </summary>
+        /// <param name="typeName">name of the Type of the Data Provider</param>
+        /// <param name="providerAttribute"> the corresponding attribute</param>
+        /// <param name="providerName"> the target provider Name</param>
+        /// <param name="configurationProfile">the target configuration profile</param>
+        /// <param name="runtimePlatform">the target runtime plaform</param>
+        protected void ApplyDataProviderConfiguration(
+            string typeName,
+            MixedRealityDataProviderAttribute providerAttribute,
+            SerializedProperty providerName,
+            SerializedProperty configurationProfile,
+            SerializedProperty runtimePlatform)
+        {
+            if (typeName != null)
+            {
+                if (providerAttribute != null)
+                {
```

Comment:
Great pulling this out to base class, but keep the DataproviderAttribute find here